docs/examples/and.rst

```diff
@@ -220,13 +220,13 @@ the problem. Its last line prints a confirmation that indeed the two selected qu
 (coupled).
 
 >>> from dwave.system.composites import FixedEmbeddingComposite
->>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})
+>>> sampler_embedded = FixedEmbeddingComposite(sampler, {'x': [0], 'z': [4]})   # doctest: +SKIP
 >>> print(sampler_embedded.adjacency)     # doctest: +SKIP
 {'x': {'z'}, 'z': {'x'}}
 
 As before, we ask for 5000 samples.
 
->>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)
+>>> response = sampler_embedded.sample_qubo(Q_not, num_reads=5000)      # doctest: +SKIP
```

Comment:
I think we should start the process of switching these over to `sampleset`

Reply:
I agree but not in this PR. I'll be going through all examples as part of my next PR to use a mock sampler in doctests and that will be the right time to update the examples, as I'll be doing that anyway to extend the doctest coverage by adjusting outputs to reliably reproducible ones